Content/Data/Modifications.Body.cs

```diff
@@ -0,0 +1,244 @@
+
+			definitions.Add(Modification.Definition.New<Body.Data>
+			(
+				identifier: "body.lightweight_design",
+				category: "Body",
+				name: "Lightweight Design",
+				description: "Redesign the object to have slightly less weight but be harder to produce",
+
+				can_add: static (ref Modification.Context context, in Body.Data data, ref Modification.Handle handle, Span<Modification.Handle> modifications) =>
+				{
+					return !modifications.HasModification(handle);
+				},
+
+				apply_1: static (ref Modification.Context context, ref Body.Data data, ref Modification.Handle handle, Span<Modification.Handle> modifications) =>
+				{
+					data.mass_multiplier *= 0.80f;
+					data.gravity *= 0.90f;
+					foreach (ref var requirement in context.requirements_new)
+					{
+						if (requirement.type == Crafting.Requirement.Type.Work)
+						{
+							requirement.amount *= 1.30f;
+							requirement.difficulty += 2.0f;
+						}
+					}
+				}
+			));
+
+			definitions.Add(Modification.Definition.New<Body.Data>
+			(
+				identifier: "body.heavyweight_design",
+				category: "Body",
+				name: "Heavyweight Design",
+				description: "Redesign the object to be a lot heavier",
+
+				apply_1: static (ref Modification.Context context, ref Body.Data data, ref Modification.Handle handle, Span<Modification.Handle> modifications) =>
+				{
+					data.mass_multiplier *= 2.00f; 
+					data.gravity *= 1.10f;
+					//This modifier is mostly a negative but sometimes you may want to increase the weight of an object like,
+					//For example you can increase the weight of your own melee weapons to make them harder to grab for your opponents
+					foreach (ref var requirement in context.requirements_new)
+					{
+						if (requirement.type == Crafting.Requirement.Type.Work)
+						{
+							requirement.amount *= 1.10f;
+						}
+					}
+				}
+			));
+
+			definitions.Add(Modification.Definition.New<Body.Data>
+			(
+				identifier: "body.floaty",
+				category: "Body",
+				name: "Floaty",
+				description: "Makes the object fall slower",
+
+				can_add: static (ref Modification.Context context, in Body.Data data, ref Modification.Handle handle, Span<Modification.Handle> modifications) =>
+				{
+					return !modifications.HasModification(handle);
+				},
+
+				apply_1: static (ref Modification.Context context, ref Body.Data data, ref Modification.Handle handle, Span<Modification.Handle> modifications) =>
+				{
+					data.gravity *= 0.20f; 
+					//Very fun modifer which makes an object behave very floaty
+					//TODO: should cost motion salt or some sort of gas, current cost is temporary
+					foreach (ref var requirement in context.requirements_new)
+					{
+						if (requirement.type == Crafting.Requirement.Type.Work)
+						{
+							requirement.amount *= 1.30f;
+						}
+					}
+					context.requirements_new.Add(Crafting.Requirement.Resource("chitin", 50));
+					 //TODO: Cost should be equal to mass but i cannot access mass right now
+				}
+			));
```

Comment:
Let's add these once there are magical essences, since this doesn't make much sense right now, physics-wise. 😉

Reply:
Done